test/SmokeTests/OOProcSmokeTests/DotNetIsolated/Counter.cs

```diff
@@ -0,0 +1,144 @@
+{
+    [Function(nameof(SignalCounter))]
+    public static async Task<HttpResponseData> SignalCounter(
+       [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "counter/{id}")] HttpRequestData request,
+       [DurableClient] DurableTaskClient client,
+       FunctionContext executionContext,
+       CancellationToken cancellation,
+       string id)
+    {
+        ILogger logger = executionContext.GetLogger(nameof(Counter));
+
+        using StreamReader reader = new StreamReader(request.Body, Encoding.UTF8);
+        string body = await reader.ReadToEndAsync();
+        if (! int.TryParse(body, out var count))
+        {
+            var httpResponse = request.CreateResponse(System.Net.HttpStatusCode.BadRequest);
+            httpResponse.Headers.Add("Content-Type", "text/plain; charset=utf-8");
+            httpResponse.WriteString($"Request body must contain an integer that indicates the number of signals to send.\n");
+            return httpResponse;
+        };
+
+        var entityId = new EntityInstanceId("Counter", id);
+        logger.LogInformation($"Sending {count} increment messages to {entityId}...");
+
+        await Parallel.ForEachAsync(
+            Enumerable.Range(0, count), 
+            cancellation, 
+            (int i, CancellationToken cancellation) =>
+            {
+                return new ValueTask(client.Entities.SignalEntityAsync(entityId, "add", 1, cancellation:cancellation));
+            });
+
+        logger.LogInformation($"Sent {count} increment messages to {entityId}.");
+        return request.CreateResponse(System.Net.HttpStatusCode.Accepted);
+    }
+
+    [Function(nameof(ReadCounter))]
+    public static async Task<HttpResponseData> ReadCounter(
+      [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "counter/{id}")] HttpRequestData request,
+      [DurableClient] DurableTaskClient client,
+      FunctionContext executionContext,
+      string id)
+    {
+        ILogger logger = executionContext.GetLogger(nameof(Counter));
+        var entityId = new EntityInstanceId("Counter", id);
+
+        logger.LogInformation($"Reading state of {entityId}...");
+        var response = await client.Entities.GetEntityAsync(entityId, includeState: true);
+        logger.LogInformation($"Read state of {entityId}: {response?.SerializedState ?? "(null: entity does not exist)"}");
+
+        if (response == null)
+        {
+            return request.CreateResponse(System.Net.HttpStatusCode.NotFound);
+        }
+        else
+        {
+            int currentValue = response.ReadStateAs<Counter>()!.CurrentValue;
+            var httpResponse = request.CreateResponse(System.Net.HttpStatusCode.OK);
+            httpResponse.Headers.Add("Content-Type", "text/plain; charset=utf-8");
+            httpResponse.WriteString($"{currentValue}\n");
+            return httpResponse;
+        }
+    }
+
+    [Function(nameof(DeleteCounter))]
+    public static async Task<HttpResponseData> DeleteCounter(
```

Comment:
Can you add a comment explaining how delete is an implicit operation.

``` CSharp
// All entities have a "delete" operation built into them.
```

Reply:
does that mean 'delete' is a reserved operation name? May be worth calling that out if so

Reply:
It's not exactly 'reserved' it just has a default implemntation already defined.